app/src/main/res/values/strings.xml

```diff
@@ -68,4 +68,6 @@
 
 
     <string name="tap_message">Tap on a message to send reply</string>
+    <string name="no_message">No Messages to receive.</string>
```

Comment:
This sounds like there are no new messages. Is that always true? If not, a better message would be: `No messages received.`

Also, `Message` should be lower case.